handlers/BusStopDepartureFinderIntentHandler.js

```diff
@@ -0,0 +1,46 @@
+const axios = require('axios');
+
+const {
+  STOP_ID_TO_NAME,
+  BUS_ID_TO_NAME,
+  SKILL_NAME
+} = require('../utilities/constants/constants');
+
+const BusStopDepartureFinderIntentHandler = {
+  canHandle(handlerInput) {
+    return handlerInput.requestEnvelope.request.type === 'IntentRequest'
+        && handlerInput.requestEnvelope.request.intent.name === 'BusStopDepartureFinderIntent';
+  },
+
+  async handle(handlerInput) {
+    // sets slot values from AWS Alexa to stopId
+    const stopId = handlerInput.requestEnvelope.request.intent.slots.STOP_ID.resolutions.resolutionsPerAuthority[0].values[0].value.id;
+    const slotId = handlerInput.requestEnvelope.request.intent.slots.BUS_ROUTE.resolutions.resolutionsPerAuthority[0].values[0].value.id
+    const stopInfo = STOP_ID_TO_NAME[stopId];
+    const busInfo = BUS_ID_TO_NAME[slotId];
+    const nextDeparture = await axios.get(`https://realtime.catabus.com/InfoPoint/rest/StopDepartures/Get/${stopInfo.stopId}`).then(response => response.data);
+    const foundBusInfoJson = nextDeparture[0].RouteDirections.find(busInfoJson => {
```

Comment:
busInfoJson is a weird variable name for this case. You are iterating over the RouteDirections array each object should probably just be called routeDirection or RouteDirection instead of busInfoJson